GrpcInterface/Python/rips/instance.py

```diff
@@ -56,6 +57,18 @@ def __is_valid_port(port):
             return False
         return True
 
+    @staticmethod
+    def __read_port_number_from_file(file_path):
+        while not os.path.exists(file_path):
```

Comment:
Probably need a loop exit if the file is not created within a resonable time here to avoid infinite loop here?

Reply:
Yes, true. Fixed now.